packages/ui/src/ui/libs/monaco-yql-languages/clickhouse/clickhouse.contribution.ts

```diff
@@ -7,24 +7,46 @@ import {
 } from '../_.contribution';
 import {languages} from 'monaco-editor';
 import {createProvideSuggestionsFunction} from '../helpers/createProvideSuggestionsFunction';
-import {parseClickHouseQuery} from '@gravity-ui/websql-autocomplete';
-import {generateClickhouseAdditionalSuggestion} from './clickhouse.keywords';
+import {
+    generateClickhouseAdditionalSuggestion,
+    generateClickhouseOldSafariSuggestions,
+} from './clickhouse.keywords';
 import {MonacoLanguage} from '../../../constants/monaco';
+import {Toaster} from '@gravity-ui/uikit';
+import {rumLogError} from '../../../rum/rum-counter';
 
 registerLanguage({
     id: MonacoLanguage.CHYT,
     extensions: [],
-    loader: () =>
-        import('./clickhouse').then((module) => {
+    loader: async () => {
+        const yql = await import('./clickhouse');
+        try {
+            //autocomplete breaks safari < 16.6
+            const autocomplete = await import('@gravity-ui/websql-autocomplete');
             return {
-                conf: module.conf,
-                language: module.language,
+                conf: yql.conf,
+                language: yql.language,
                 provideSuggestionsFunction: createProvideSuggestionsFunction(
-                    parseClickHouseQuery,
+                    autocomplete.parseClickHouseQuery,
                     generateClickhouseAdditionalSuggestion,
                 ),
             };
-        }),
+        } catch (e) {
+            const toaster = new Toaster();
+            toaster.add({
+                type: 'error',
+                name: 'Autocomplete clickhouse error',
```

Comment:
As I know name is never displayed, it is used inside Toaster to split toasts from different parts of ui, i.e. your current `name` looks like a title, let's provide more details like:
```ts
name: 'clickhouse_autocomplete',
title: 'Autocomplete error',
content: 'Your browser does not support all required features, it will work in restricted mode.'
```